packages/desktop-activity/src/lib/kb-mouse-timer.ts

```
@@ -12,7 +12,9 @@ export class KbMouseTimer {
 	private afkDuration = 0;
 	private afkCountdown: number;
 	private isAfk = false;
-	private onAfkCallback: (() => void) | null = null;
+	private onAfkCallback: ((afk?: boolean) => void) | null = null;
+	private isStarted:boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)